.travis.yml

```diff
@@ -18,6 +18,7 @@ git:
 
 matrix:
     fast_finish: true
+    fail_fast: true
```

Comment:
I actually don't think we should have this and leave it false. That way you can see the full gamut of errors (of course for your testsing ok)